wsd/ClientSession.cpp

```diff
@@ -690,6 +690,11 @@ bool ClientSession::_handleInput(const char *buffer, int length)
                                 << "] has no write permissions in Storage and cannot save.");
             sendTextFrameAndLogError("error: cmd=save kind=savefailed");
         }
+        else if(_wopiFileInfo && _wopiFileInfo->getHideSaveOption())
```

Comment:
Please use clang-format or manually format code to be consistent with C++ code in the project.

There is always a space after `if` tokens.

Reply:
yeah thanks for noticing, I just switched editor I will configure clang-format, which clang formatting style do we use ?

Reply:
before this I was using vscode and it was falling back to its own style
 https://code.visualstudio.com/docs/cpp/cpp-ide